.github/workflows/dev-deploy-cloud-functions.yml

```
@@ -0,0 +1,79 @@
+    - name: Set up gcloud CLI
+      uses: google-github-actions/setup-gcloud@v2
+
+    - name: Deploy Function
+      if: needs.detect_changes.outputs[matrix.function] == 'true'
+      run: |
+        gcloud functions deploy ${{ matrix.function }} \
+          --gen2 \
+          --region=us-west1 \
+          --runtime=python310 \
+          --source=cloud_functions/${{ matrix.function }} \
+          --trigger-http
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)